week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +51,16 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for(let i = 0; i < arr.length; i++){
+    arr[i] = Number(arr[i].toFixed(2));
+    if(arr[i] > 100)
+    arr[i] = 100;
```

Comment:
Please pay more attention to how to ident the code, it helps the reader to understand easier.
Easy to miss that code in row 58 belongs to the if statement above it.